solutions/fully-configurable/variables.tf

```diff
@@ -0,0 +1,93 @@
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API key used to provision resources."
+  sensitive   = true
+}
+
+variable "existing_secrets_manager_crn" {
+  type        = string
+  description = "CRN of an existing secrets manager instance to create the secret engine in."
+}
+
+variable "prefix" {
+  type        = string
+  description = "The prefix to be added to all resources created by this solution. To skip using a prefix, set this value to null or an empty string. The prefix must begin with a lowercase letter and may contain only lowercase letters, digits, and hyphens '-'. It should not exceed 16 characters, must not end with a hyphen('-'), and can not contain consecutive hyphens ('--'). Example: prod-us-south."
```

Comment:
this is missing the link to the new helper doc: https://terraform-ibm-modules.github.io/documentation/#/prefix.md

Please follow these [guidelines](https://terraform-ibm-modules.github.io/documentation/#/da-implementation-guidelines?id=code-snippet-example) for prefix variable